src/main/java/liquibase/ext/opensearch/integration/spring/LiquibaseOpenSearchAutoConfiguration.java

```
@@ -0,0 +1,26 @@
+package liquibase.ext.opensearch.integration.spring;
+
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.springframework.aot.hint.RuntimeHints;
+import org.springframework.aot.hint.RuntimeHintsRegistrar;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ImportRuntimeHints;
+
+@AutoConfiguration
+@EnableConfigurationProperties(SpringLiquibaseOpenSearchProperties.class)
+@ImportRuntimeHints(LiquibaseOpenSearchAutoConfiguration.LiquibaseAutoConfigurationRuntimeHints.class)
```

`@ConditionalOnBean(OpenSearchClient.class)` ?

---

no, i expect it to be present. otherwise it won't work and why would you then pull in `liquibase-opensearch-spring-boot-starter`? if i'd do `@ConditionalOnBean` it'd silently fail, this way it fails in a noisy and obvious way.

unless it's a requirement that you can pull in the dependency without having it work? 🤔